pkg/pod/pod.go

```diff
@@ -169,6 +173,27 @@ func (builder *Builder) DefineOnNode(nodeName string) *Builder {
 	return builder
 }
 
+// Get returns a Pod object if found.
+func (builder *Builder) Get() (*corev1.Pod, error) {
```

Comment:
Pod is standard package so we don't need Get function because Exist function provides this functionality. You can remove Get